<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>940</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei Light</family>
    <pointsize>-1</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>中国象棋</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
   font-family:&quot;Microsoft YaHei Light&quot;;
   font-size:18px;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>980</width>
     <height>920</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
    background-color: transparent;
    border-image: url(:/Images/bg.jpg);
    border-radius:6px;
    opacity:0.5;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Icon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessIcon.ico);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:40px</string>
    </property>
    <property name="text">
     <string>中国象棋 ChineseChess</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MinButton">
    <property name="geometry">
     <rect>
      <x>887</x>
      <y>0</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">#MinButton{
   border-image:url(:/Images/min.ico);
   border-radius:5px;
}
#MinButton:pressed{
   border-image:url(:/Images/min_press.bmp);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CloseButton">
    <property name="geometry">
     <rect>
      <x>932</x>
      <y>0</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#CloseButton{
   border-image:url(:/Images/close.ico);
   border-radius:5px;
}

#CloseButton:pressed{
   border-image:url(:/Images/close_press.ico);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ChessBoard">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>741</width>
      <height>821</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/BOARD.BMP);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="BlackChe1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/br.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>b</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="BlackMa1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/bn.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>b</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="BlackXiang1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/bb.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>b</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="BlackShi1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/ba.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>b</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="BlackJiang">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/bk.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>b</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="BlackShi2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/ba.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>b</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="BlackXiang2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/bb.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>b</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="BlackMa2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>90</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/bn.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>b</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="BlackChe2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>90</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/br.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>b</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="BlackPao1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/bc.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>b</string>
    </property>
   </widget>
   <widget class="QLabel" name="BlackPao2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>250</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/bc.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>b</string>
    </property>
   </widget>
   <widget class="QLabel" name="BlackBing1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/bp.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>b</string>
    </property>
   </widget>
   <widget class="QLabel" name="BlackBing2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/bp.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>b</string>
    </property>
   </widget>
   <widget class="QLabel" name="BlackBing3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>330</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/bp.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>b</string>
    </property>
   </widget>
   <widget class="QLabel" name="BlackBing4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>330</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/bp.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>b</string>
    </property>
   </widget>
   <widget class="QLabel" name="BlackBing5">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>330</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/bp.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>b</string>
    </property>
   </widget>
   <widget class="QLabel" name="RedBing1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/rp.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QLabel" name="RedBing2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>570</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/rp.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QLabel" name="RedBing3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>570</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/rp.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QLabel" name="RedBing4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>570</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/rp.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QLabel" name="RedBing5">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>570</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/rp.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QLabel" name="RedPao1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>650</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/rc.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QLabel" name="RedPao2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>650</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/rc.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>r</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="RedChe1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>810</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/rr.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QLabel" name="RedXiang1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>810</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/rb.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QLabel" name="RedMa1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>810</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/rn.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QLabel" name="RedShi1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>810</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/ra.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QLabel" name="RedJiang">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>810</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/rk.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QLabel" name="RedShi2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>810</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/ra.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QLabel" name="RedXiang2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>810</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/rb.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QLabel" name="RedChe2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>810</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/rr.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QLabel" name="RedMa2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>810</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/rn.ico);
color:transparent;</string>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="ComputerScoreBox">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>70</y>
      <width>201</width>
      <height>351</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>中国象棋云库</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="ComputerScore">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:40px;
color:green;</string>
     </property>
     <property name="text">
      <string>云库出步</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="BookLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>181</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:22px</string>
     </property>
     <property name="text">
      <string>默认由ChessDB提供的中国象棋云库出步。

云库无对应走法或获取走法失败时由引擎出步。</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="SideSelectionBox">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>440</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>红黑方选择</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QComboBox" name="PlayerSide">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:40px</string>
     </property>
     <item>
      <property name="text">
       <string>红</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>黑</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="PlayerSideLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:40px</string>
     </property>
     <property name="text">
      <string>我执</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Reset">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>720</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:30px;</string>
    </property>
    <property name="text">
     <string>😀 重置棋盘</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Flip">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>820</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:30px;</string>
    </property>
    <property name="text">
     <string>🔄 翻转棋盘</string>
    </property>
   </widget>
   <widget class="QLabel" name="Mask1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>450</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/mask2.png);
color:transparent;</string>
    </property>
    <property name="text">
     <string>b</string>
    </property>
   </widget>
   <widget class="QLabel" name="Mask2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>450</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/mask.png);
color:transparent;</string>
    </property>
    <property name="text">
     <string>b</string>
    </property>
   </widget>
   <widget class="QLabel" name="Mask3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>450</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ChessImage/mask.png);
color:transparent;</string>
    </property>
    <property name="text">
     <string>b</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="HardSelectionBox">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>570</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>电脑难度选择</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QComboBox" name="ComputerHard">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:40px</string>
     </property>
     <item>
      <property name="text">
       <string>    简单</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>    普通</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>    地狱</string>
      </property>
     </item>
    </widget>
   </widget>
   <zorder>Icon</zorder>
   <zorder>Title</zorder>
   <zorder>MinButton</zorder>
   <zorder>CloseButton</zorder>
   <zorder>ChessBoard</zorder>
   <zorder>BlackChe1</zorder>
   <zorder>BlackMa1</zorder>
   <zorder>BlackXiang1</zorder>
   <zorder>BlackShi1</zorder>
   <zorder>BlackJiang</zorder>
   <zorder>BlackShi2</zorder>
   <zorder>BlackXiang2</zorder>
   <zorder>BlackMa2</zorder>
   <zorder>BlackChe2</zorder>
   <zorder>BlackPao1</zorder>
   <zorder>BlackPao2</zorder>
   <zorder>BlackBing1</zorder>
   <zorder>BlackBing2</zorder>
   <zorder>BlackBing3</zorder>
   <zorder>BlackBing4</zorder>
   <zorder>BlackBing5</zorder>
   <zorder>RedBing1</zorder>
   <zorder>RedBing2</zorder>
   <zorder>RedBing3</zorder>
   <zorder>RedBing4</zorder>
   <zorder>RedBing5</zorder>
   <zorder>RedPao1</zorder>
   <zorder>RedPao2</zorder>
   <zorder>RedChe1</zorder>
   <zorder>RedXiang1</zorder>
   <zorder>RedMa1</zorder>
   <zorder>RedShi1</zorder>
   <zorder>RedJiang</zorder>
   <zorder>RedShi2</zorder>
   <zorder>RedXiang2</zorder>
   <zorder>RedChe2</zorder>
   <zorder>RedMa2</zorder>
   <zorder>ComputerScoreBox</zorder>
   <zorder>SideSelectionBox</zorder>
   <zorder>Flip</zorder>
   <zorder>Reset</zorder>
   <zorder>Mask1</zorder>
   <zorder>Mask2</zorder>
   <zorder>Mask3</zorder>
   <zorder>HardSelectionBox</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
